cli/Sources/TuistServer/Models/ServerProject.swift

```diff
@@ -42,6 +42,12 @@ extension ServerProject {
     }
 }
 
+extension ServerProject: CustomStringConvertible {
```

Comment:
Why not defining the conformance in the body of `ServerProject` to make things easier for the compiler?